docs/permalinks.md

```diff
@@ -0,0 +1,10 @@
+# Permalink
+
+Altis introduces a fix to a bug where the main site on a network is hardcoded to contain `/blog/` prefix for posts by default. The side-effect for this is that you cannot use `/blog/` prefix on the main site unless you disable the fix, which is by unhooking the filters associated with it, eg:
+
+```php
+remove_filter( 'sanitize_option_permalink_structure', '\Altis\CMS\Permalinks\remove_blog_prefix' );
+remove_filter( 'option_permalink_structure', '\Altis\CMS\Permalinks\remove_blog_prefix' );
```

Comment:
I think it'll save us some potential support to wrap these in an action so we know they're being removed after they've been added, I think `plugins_loaded` would be fine:

```suggestion
add_action( 'plugins_loaded', function () {
    remove_filter( 'sanitize_option_permalink_structure', '\Altis\CMS\Permalinks\remove_blog_prefix' );
    remove_filter( 'option_permalink_structure', '\Altis\CMS\Permalinks\remove_blog_prefix' );
} );
```